data/src/main/java/pl/edu/agh/model/Notification.java

```diff
@@ -1,5 +1,41 @@
 package pl.edu.agh.model;
 
+import java.time.LocalDateTime;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Transient;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
 public class Notification {
 
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
Tu bym dała tu też komentarz `// TODO:`, skoro tego idka mamy w `BaseEntity`. Ewentualnie można też się już tutaj zrebasować na `IO2019TEAM-28` i korzystać z dobrodziejstw (tylko jak @szbrnl tam będzie amendował do commita, to trzeba pamiętać, że im się hashe zmieniają, bo to lubi zaskoczyć)

Reply:
może po prostu poczekać, aż -28 się pojawi na masterze